incubator/hnc/config/crd/bases/hnc.x-k8s.io_hierarchies.yaml

```diff
@@ -52,17 +52,56 @@ spec:
                 type: string
               type: array
             conditions:
+              description: Conditions describes the errors and the affected objects,
+                if any.
               items:
+                description: Condition specifies the condition and the affected objects.
                 properties:
+                  affects:
+                    description: Affects is a list of group-version-kind-namespace-name
+                      that uniquely identifies the object(s) affected by the condition.
+                    items:
+                      description: AffectedObject defines uniquely identifiable objects.
+                      properties:
+                        group:
+                          type: string
+                        kind:
+                          type: string
+                        name:
+                          type: string
+                        namespace:
+                          type: string
+                        version:
+                          type: string
+                      type: object
+                    type: array
+                  code:
+                    description: 'Defines the conditions in a machine-readable string
+                      value. Valid values are: - "CRIT_PARENT_MISSING": the specified
```

Comment:
I'm noticing that all your newlines seem to be removed here. Perhaps adding a blank line between each of the bullet points in the comment will help?

Reply:
I added blank lines in the comments and did see the newlines "/n" here. However, I checked my previous commits, there were no newlines either. https://github.com/yiqigao217/multi-tenancy/commit/d252f90663a768661b7ac4f470b13bd3bfa36bc4

Reply:
That's fine if this wasn't looking good before, let's just make it better now :) Thanks!